modules/mesos-cluster/main.tf

```diff
@@ -76,7 +80,7 @@ resource "aws_launch_configuration" "mesos-master-ec2-lc" {
 
 resource "aws_launch_configuration" "mesos-slave-ec2-lc" {
   name_prefix                 = "mesos-slave-${var.environment}-lc-"
-  image_id                    = "${lookup(var.aws_amis, var.aws_region)}"
+  image_id                    = "${var.environment == "production" ? lookup(var.aws_amis, var.aws_region) : lookup(var.aws_amis, format("%s-%s", var.aws_region, "ubuntu-16-04"))}"
```

Comment:
I _really_ want to keep AMI IDs in the variables file otherwise we're going to forget about some.

The lookup and format doesn't look pretty but it'll be temporary until we use the same AMI again.